frontend/lib/norent/components/subscribe.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { Trans, t } from "@lingui/macro";
+import { li18n } from "../../i18n-lingui";
+import { StaticImage } from "../../ui/static-image";
+import { getImageSrc } from "../homepage";
+
+type SubscribeProps = {};
+
+type SubscribeState = {
+  email: string;
+  success: boolean;
+  response: string;
+};
+
+class Subscribe extends React.Component<SubscribeProps, SubscribeState> {
+  constructor(props: SubscribeProps) {
+    super(props);
+    this.state = {
+      email: "",
+      success: false,
+      response: "",
+    };
+  }
+
+  handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({ email: e.target.value });
+  };
+
+  handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const email = this.state.email || null;
+    const locale = li18n.language;
+
+    if (!email) {
+      this.setState({
+        response: li18n._(t`Please enter an email address!`),
+      });
+      return;
+    }
+
+    fetch("/mailchimp/subscribe", {
```

Comment:
err, can you use `awesomeFetch` here instead?  It's the exact same thing but it basically loads a fetch polyfill if needed (i.e., IE11).  Otherwise this sadly will break on IE11.

Reply:
aWeSoMEfeTCh!!